cli/pkg/dotrillcloud/dotrillcloud.go

```diff
@@ -0,0 +1,45 @@
+package dotrillcloud
+
+import (
+	"os"
+	"path"
+
+	"gopkg.in/yaml.v3"
+)
+
+type RillCloud struct {
+	ProjectID string `json:"project_id"`
+}
+
+const confPath = ".rillcloud/project.yaml"
+
+func GetAll(localProjectPath string) (*RillCloud, bool, error) {
```

Comment:
The value of returned `bool` is always false. Is that expected ?
Also why do we need the bool ?

Reply:
No this is a remnant of some old code. Removing it